cvat/apps/engine/models.py

```diff
@@ -223,6 +223,7 @@ class Image(models.Model):
     frame = models.PositiveIntegerField()
     width = models.PositiveIntegerField()
     height = models.PositiveIntegerField()
+    orientation = models.PositiveIntegerField(default=1)
```

Comment:
Do you mean we should store rotated images on the server side? In that case, there would be a mismatch in the resolution of the raw data and compressed. I don't see a big problem here, but the whole idea behind this PR was to try to not modify the original data if it's possible.

Reply:
Maybe more general solution is to store received EXIF information as a JSON in our database (as a part of meta information), instead of storing only orientation?

Reply:
I'm not sure about this. What kind of information can be helpful for us? I think we can move to that solution if we will have such a request to parse and use other EXIF tags. But EXIF usually stores a lot of information like thumbnail and photo instruments info which is quite useless for us but costs a lot of storage.